openquake/commonlib/valid.py

```diff
@@ -670,21 +670,22 @@ def ab_values(value):
 vs30_type = ChoiceCI('measured', 'inferred')
 
 SiteParam = collections.namedtuple(
-    'SiteParam', 'z1pt0 z2pt5 measured vs30 lon lat'.split())
+    'SiteParam', 'z1pt0 z2pt5 measured vs30 backarc lon lat'.split())
 
```

Comment:
Let's move backarc to the last position, to be consistent with the signature of the function site_param below.